docs/README.md

```diff
@@ -1044,6 +1044,140 @@ docker-compose up
 2. Site will be avaiable on `localhost:3001`
 3. You can update the documentation by modifying the file `docs/README.md`
 
+## OpReturn
+
+OP_RETURN is a [script opcode](https://wiki.bitcoinsv.io/index.php/OP_RETURN) that allows adding extra information to transactions beyond standard inputs and outputs. This can be useful for applications that want to include:
+
+* Different transaction types related to events (e.g., gaming);
```

Comment:
the example should contain something else to be about **different types**, 'gaming' is only one type so maybe put here: `(e.g. 'vegetables', 'coffee', 'cleaning products', etc.)`

could be other stuff related to "gaming" I just couldn't think of anything that made for good examples of "categories of the same thing"



(the comma after e.g. is also wrong)